src/services/place.ts

```diff
@@ -86,10 +86,7 @@ export default class Place {
 
     for (const hierarchyLevel of Config.getHierarchyWithReplacement(this.type)) {
       const propertyName = hierarchyLevel.property_name;
-      delete this.hierarchyProperties[propertyName];
```

Comment:
Here is a scenario which is breaking due to this change:

1. Create a Sub-County `A` 
3. Create a CHU `AA` in sub-county `A`
4. Upload all to CHT

Then:
1. Create a CHP setting Sub-County to empty and CHU to `AA`. This will work.
2. Now edit the CHP and set sub-county to A and CHU to AA. This will work.
3. Edit the CHP and change it back to state 1 by setting Sub-County to empty.  

This edit scenario works in `main` and is broken due to your change.
I think we should have a test to capture this, because it isn't totally obvious.

Reply:
Why is this required here but not when we create places from csv https://github.com/medic/cht-user-management/blob/dc033f33578689384bfb4a52484ed4e9e8d46911/src/services/place-factory.ts#L69-L72? I deleted it so the fields populated are the same via csv or the UI otherwise we'd have missing columns in the final credential csvs if a user used both UI and CSV when creating users

Reply:
I reproduced the above steps on Sat when I sent the feedback; but when debugging today to answer this, I'm not able to reproduce it again....  Seems we can maybe keep as-is in this PR. Sorry, its possible I was simply mistaken.